### PR TITLE
PLU-228: [M365-GA] Add whats new message for m365 excel

### PR DESCRIPTION
**Pull request by ogp-weeloong**
## Problem
We need a what's new message for Excel.

## Solution
Screenshot 2024-06-20 at 6 16 19 PM

## Tests
- Check that what's new message for excel shows up

### CODE REVIEW

File: packages/frontend/src/components/NewsDrawer/NewsItemList.ts [hunk 1]
```diff
@@ -8,6 +8,15 @@ const IF_THEN_EXTERNAL_LINK =
   'https://guide.plumber.gov.sg/user-guides/actions/toolbox'
 
 export const NEWS_ITEM_LIST: NewsItemProps[] = [
+  {
+    date: '2024-06-24',
+    tag: NEW_ENHANCEMENT_TAG,
```
Comment (pregnantboy) [inline packages/frontend/src/components/NewsDrawer/NewsItemList.ts]: NEW_FEATURE_TAG

**Review by pregnantboy — APPROVED** [review]

File: packages/frontend/src/components/NewsDrawer/NewsItemList.ts (hunk 1 above)
Comment (ogp-weeloong) [reply packages/frontend/src/components/NewsDrawer/NewsItemList.ts]: derp, thanks for catching

**Comment by ogp-weeloong** [comment]
* **#639** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/639?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#640** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/640?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#637** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/637?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ogp-weeloong and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ogp-weeloong** [comment]
### Merge activity

* **Jun 25, 2:38 AM EDT**: @ogp-weeloong started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/637).
* **Jun 25, 2:38 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/637) rebased this pull request as part of a merge.
* **Jun 25, 2:39 AM EDT**: @ogp-weeloong merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opengovsg/plumber/637).

**Pull request merged**